src/pedal.cpp

```diff
@@ -46,4 +50,41 @@ void Pedal::Reset()
 // Pedal functor methods
 //----------------------------------------------------------------------------
 
+int Pedal::GenerateMIDI(FunctorParams * functorParams)
+{
+    GenerateMIDIParams *params = dynamic_cast<GenerateMIDIParams *>(functorParams);
+    assert(params);
+
+    if (!HasDir()) return FUNCTOR_CONTINUE;
+
+    int velocity;
+    int tickOffset = 0;
+    switch (GetDir())
```

Comment:
See the coding guidelines and the clang-format use for {} layout (https://github.com/rism-ch/verovio/blob/master/doc/guidelines.md)